packages/core/src/badge/Badge.tsx

```diff
@@ -41,7 +41,9 @@ export const Badge = forwardRef<HTMLSpanElement, BadgeProps>(function Badge(
       {children}
       <span
         className={clsx(withBaseName("badge"), {
-          [withBaseName("topRight")]: children,
+          [withBaseName("topRight")]: children && value,
+          [withBaseName("dotBadge")]: !value,
+          [withBaseName("topRightCenter")]: children && !value,
```

Comment:
Is it possible to just use topRight?

.topRight and .dotBadge.topRight ?

Reply:
done